django/contrib/contenttypes/management/__init__.py

```diff
@@ -77,7 +77,9 @@ def inject_rename_contenttypes_operations(plan=None, apps=global_apps, using=DEF
         for index, operation in enumerate(migration.operations):
             if isinstance(operation, migrations.RenameModel):
                 operation = RenameContentType(
-                    migration.app_label, operation.old_name_lower, operation.new_name_lower
```

Comment:
Please remove the added trailing comma -- there's little usefulness in making a change like that.